src/main/scala/com/twitter/finagle/postgres/values/JSONB.scala

```diff
@@ -0,0 +1,10 @@
+package com.twitter.finagle.postgres.values
+
+object JSONB {
+  def apply(string: String): JSONB = new JSONB(string.getBytes())
+
+  def stringify(jsonb: JSONB): String = new String(jsonb.bytes)
```

Comment:
At this point, you are just using the `Charset.defaultCharset()` to create the string right? I would say it's either that or add an optional Charset value to stringify with `defaultCharset()` or change `JSONB` be:
```
case class JSONB(str: Array[Byte])
```
and the encoder to something like: 
```
     (j, c) => {
       val cb = ChannelBuffers.buffer(1 + j.bytes.length)
       val cb: ChannelBuffer = ChannelBuffers.buffer(1 + j.value.getBytes.length)
       cb.writeByte(1)
       cb.writeBytes(j.bytes)
       cb.writeBytes(j.value.getBytes)
       Some(cb)
      }
    )
```

Reply:
Good point. Option added. Thanks!